Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -405,6 +405,9 @@ theorem norm_integral_le_integral_norm (f : α →ₛ E) (hf : Integrable f μ)
   refine (norm_setToSimpleFunc_le_integral_norm _ (fun s _ _ => ?_) hf).trans (one_mul _).le
   exact (norm_weightedSMul_le s).trans (one_mul _).symm.le
 
+lemma abs_integral_le_integral_abs {f : α → ℝ} : |∫ a, f a ∂μ| ≤ ∫ a, |f a| ∂μ :=
+  norm_integral_le_integral_norm f
```

Comment:
I'm confused why you're not obliged to pass the `hf` that line 403 says is needed.

Reply:
because you're looking at the wrong `norm_integral_le_integral_norm` lemma (because I put my new lemma in the wrong place). This one is `SimpleFunc.norm_integral_le_integral_norm` while I am using `norm_integral_le_integral_norm`